app/views/documents/new.html.erb

```diff
@@ -16,9 +16,42 @@
 
     <h2 class="govuk-heading-m">Upload new document</h2>
 
-    <%= form_with model: [@planning_application, @document], local: true, class: "document" do |form| %>
+    <%= form_with model: [@planning_application, @document], local: true, class: "document", builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
       <%= render partial: 'documents/form_partials/upload.html.erb', locals: {form: form} %>
-      <%= render partial: 'documents/form_partials/tags.html.erb', locals: {form: form} %>
+
+      <div class="govuk-form-group <%= @document.errors[:numbers].any? ? 'govuk-form-group--error' : '' %>">
+        <% if @document.errors.any? %>
+          <% @document.errors.each do |error| %>
+            <span id="status-error" class="govuk-error-message">
+              <span class="govuk-visually-hidden">Error:</span><%= error %></span>
+          <% end %>
+        <% end %>
+        <%= form.label :numbers, "Document number(s)", class: "govuk-label", for: "numbers" %>
+        <div id="event-name-hint" class="govuk-hint">
+          Where documents contain multiple drawings, enter multiple numbers separated by a comma (e.g. "25A-V2, 25B-V2").
+        </div>
+        <div class="numbers">
+          <%= form.text_area :numbers, class: "govuk-input govuk-input--width-20", id: "numbers" %>
+        </div>
+        <%= render partial: 'documents/form_partials/tags.html.erb', locals: {form: form} %>
+
+        <div class="display">
+          <%= form.govuk_radio_buttons_fieldset(:status, legend: { size: 's', text: 'Do you want to list this document on the decision notice?' }) do %>
+            <%= form.govuk_radio_button :referenced_in_decision_notice, 'true', label: { text: 'Yes' } %>
+            <%= form.govuk_radio_button :referenced_in_decision_notice, 'false', label: { text: 'No' } %>
+          <% end %>
+        </div>
+
+        <div class="publish">
+          <%= form.govuk_radio_buttons_fieldset(:status, legend: { size: 's', text: 'Should this document be made publicly available?' }) do %>
+            <div id="event-name-hint" class="govuk-hint">
+              Please indicate whether this document should be made available for public viewing on the planning register or via the public API. Select "No" if the document contains any sensitive personal information
+            </div>
+            <%= form.govuk_radio_button :publishable, 'true', label: { text: 'Yes' } %>
+            <%= form.govuk_radio_button :publishable, 'false', label: { text: 'No' } %>
+          <% end %>
+        </div>
+      </div>
```

Comment:
The first A/C on the ticket is "When a new document is uploaded or replaced the user should see the same options as when they edit a document including add drawing number and the two questions to be added in this story." which if I recall was our way of saying that the edit and new actions can just use the same form partial?